Tests/Behaviors/SnapToPointLayoutBehaviorTests.swift

```diff
@@ -356,6 +356,54 @@ class SnapToPointLayoutBehaviorTests: XCTestCase {
 
     }
 
+    func testAttributesFilter() {
+        let behavior = SnapToPointLayoutBehavior(scrollDirection: .Horizontal(.Center))
 
+        let layout = UICollectionViewLayout()
+        behavior.getFilteredAttributes(layout: layout, frame: CGRect(x: 0, y: 0, width: 480, height: 360))
 
+
+        let collection = CollectionViewController()
```

Comment:
There is also in `CollectionBrickCellModel` a `configureHandler` that you can use to add the behavior